### ISSUE TITLE
Does this thing work on Windows?

### ISSUE THREAD

**Issue by claym** [griffio/spring-security-auth0-sample#3]
I get this error when running the bundle command:

```
C:\repo\spring-security-auth0-sample>npm run bundle

> spring-security-auth0-sample@1.0.0 bundle C:\repo\spring-security-auth0-sample
> browserify --entry src/main/resources/static/index.js --outfile src/main/resources/static/bundle.js --debug

SyntaxError: Unexpected token (1:0) while parsing file: C:\repo\spring-security-auth0-sample\node_modules\auth0-lock\lib\html\main.ejs
```

**Comment by griffio** [comment]
Hey @claym,

I think the issue is with browersify, npm version combination you have installed.

See this issue for details: https://github.com/auth0/lock/issues/225

I am using npm v3.8.7 and node v4.4.3

Check your versions of node/npm and update.

Let me know if that works and I will add a note on the readme
Cheers!

**Comment by claym** [comment]
Updated to latest; [debug log gist](https://gist.github.com/claym/cfae2a34bf1101a8ec72f0c49df535f7)

```
C:\repo\spring-security-auth0-sample>node -v
v6.2.0

C:\repo\spring-security-auth0-sample>npm -v
3.9.5

C:\repo\spring-security-auth0-sample>npm install

C:\repo\spring-security-auth0-sample>npm run bundle

> spring-security-auth0-sample@1.0.0 bundle C:\repo\spring-security-auth0-sample
> browserify --entry src/main/resources/static/index.js --outfile src/main/resources/static/bundle.js --debug

SyntaxError: Unexpected token (1:0) while parsing file: C:\repo\spring-security-auth0-sample\node_modules\auth0-lock\lib\html\main.ejs
    at Parser.pp.raise (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:1745:13)
    at Parser.pp.unexpected (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:2264:8)
    at Parser.pp.parseExprAtom (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:1141:12)
    at Parser.pp.parseExprSubscripts (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:1023:19)
    at Parser.pp.parseMaybeUnary (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:1004:19)
    at Parser.pp.parseExprOps (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:948:19)
    at Parser.pp.parseMaybeConditional (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:930:19)
    at Parser.pp.parseMaybeAssign (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:908:19)
    at Parser.pp.parseExpression (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:882:19)
    at Parser.pp.parseStatement (C:\repo\spring-security-auth0-sample\node_modules\acorn\dist\acorn.js:2461:23)

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Clay\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "bundle"
npm ERR! node v6.2.0
npm ERR! npm  v3.9.5
npm ERR! code ELIFECYCLE
npm ERR! spring-security-auth0-sample@1.0.0 bundle: `browserify --entry src/main/resources/static/index.js --outfile src/main/resources/static/bundle.js --debug`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the spring-security-auth0-sample@1.0.0 bundle script 'browserify --entry src/main/resources/static/index.js --outfile src/main/resources/static/bundle.js --debug'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the spring-security-auth0-sample package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     browserify --entry src/main/resources/static/index.js --outfile src/main/resources/static/bundle.js --debug
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs spring-security-auth0-sample
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls spring-security-auth0-sample
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\repo\spring-security-auth0-sample\npm-debug.log
```

**Comment by griffio** [comment]
Hey @claym,

Firstly, I updated some out of date npm packages in the repo package.json

To test it out on Windows, I setup a clean install with Node,npm on a Virtual Machine with Win8.1

Steps that verify. 

Installed [latest](https://nodejs.org/en/download/current/), v6.2.1 via .msi, includes npm v3.9.3

Then, installed latest npm v3.9.5 using command "npm install npm@latest -g"

-> npm@3.9.5 C:\Users\Clay\AppData\Roaming\npm\node_modules\npm

Make sure the Windows PATH has %APPDATA%\npm at the front. Removed user PATH variable then added %APPDATA%\npm to the System PATH variable at the front.

Worth checking your path doesn't have other versions of npm.
Should only have these in your system path, with npm before nodejs in the PATH.
-> ;C:\Users\Clay\AppData\Roaming\npm;C:\Program Files\nodejs\

Finally,

git clone https://github.com/griffio/spring-security-auth0-sample.git

cd spring-security-auth0-sample 

npm install
...zzz
npm run bundle

That worked for me!

Sorry if it still doesn't work, don't know how else to help.
